vertical-pod-autoscaler/recommender/main.go

```diff
@@ -22,22 +22,23 @@ import (
 
 	"github.com/golang/glog"
 	kube_flag "k8s.io/apiserver/pkg/util/flag"
+	"k8s.io/autoscaler/vertical-pod-autoscaler/recommender/signals"
 	"k8s.io/client-go/rest"
 	kube_restclient "k8s.io/client-go/rest"
 )
 
 var (
-	namespace              = *flag.String("namespace", "default", `Namespace to manage`)
-	metricsFetcherInterval = *flag.Duration("recommender-interval", 1*time.Minute, `How often metrics should be fetched`)
-	prometheusAddress      = *flag.String("prometheus-address", "", `Where to reach for Prometheus metrics`)
+	namespace              = flag.String("namespace", "default", `Namespace to manage`)
```

Comment:
namespace of what? VPA recommenders are expected to be global.

Reply:
Completely agree, but it's not the scope of this change. The flags were already here, I just noticed that with the star here it doesn't work (takes the default value before initializing the flag with value from the command line).

Reply:
ok. Please add a todo to fix it.

Reply:
Done